scaffoldmaker/utils/eftfactory_tricubichermite.py

```diff
@@ -100,21 +100,21 @@ def createEftShellPole90(self, quadrant):
         assert eft.validate(), 'eftfactory_tricubichermite.createEftShellPole90:  Failed to validate eft'
         return eft
 
-    def createEftShellApexBottom(self, nodeScaleFactorOffset0, nodeScaleFactorOffset1):
```

Comment:
Very good that you changed the name... please search through other mesh types to ensure they use the new name. I'm sure there are a couple.